gateway-service/src/main/resources/configs/common/application.conf

```diff
@@ -87,3 +87,8 @@ scopeFiltersConfig = [
     ]
   }
 ]
+
+log.config = {
```

Comment:
do we need to add it to helm templates ?

Reply:
Checked helm template, and it has the application.conf as a configmap. So, if it needs to be overriden, we only need to override it in helm-values.yaml.